.github/workflows/erlang.yml

```diff
@@ -9,33 +10,41 @@ jobs:
 
   build:
 
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-22.04
 
     strategy:
       matrix:
-        otp: ['25.2.1']
-        rebar: ['3.20.0']
+        otp: ["25", "26"]
```

Comment:
We let `erlef/setup-beam` decide on the latest version in the major branch, thus saving some maintenance time.